src/test/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksVendorSupplierDiversityServiceImplTest.java

```diff
@@ -80,56 +80,56 @@ private void addSUpplierDiversityToMap(List<KfsToPMWSupplierDiversityDTO> suppli
     }
 
     @Test
-    public void testBuildSuppplierDivsersityListFromPaymentWorksVendorWithRealExamples() {
-        pmwVendor.setFederalDivsersityClassifications(FEDERAL_CLASSIFICAITONS);
-        pmwVendor.setStateDivsersityClassifications(STATE_CLASSIFICAITONS);
+    public void testBuildSuppplierDiversityListFromPaymentWorksVendorWithRealExamples() {
```

Comment:
The various test methods in this class have an extra-"p" typo that still needs to be fixed.

Reply:
ok